template.tpl

```diff
@@ -924,15 +958,10 @@ const getReferenceIdx = (entity, refsList) => {
  * @returns {Object[]} The valid rules
  */
 const cleanRules = (rules) => {
+  const lastNumRexp = createRegex('[0-9]$', 'g');
```

Comment:
Should this use the same `_[0-9]+$` pattern as in `mkVersionFree`? Maybe create it once globally and reuse it in both functions?

Reply:
The `cleanRules` function is checking the user provided `key`, i.e. the entity schema, which can also be specified with its original `iglu` URI. Still a pretty naive check, but does not address only for the enriched schema name, so an underscore is not necessary for the input to be correct.

Reply:
We could possibly go for something stricter e.g. `createRegex('[_-][0-9]+$');`. Still a different regexp, but more explicit than it is now. What do you think?

Reply:
It would probably need to check for a full version string to be worth it in the Iglu URI case, so it's OK for now I think. Thanks!